examples/auto-scaling/rstudio-workbench/users.txt

```diff
@@ -0,0 +1,5 @@
+username:uid:User, Name
```

Comment:
I like this! One note - Is `User, Name` or `uid` used in the user creation?

Also, it might be worth setting the password in this file, rather than just setting to the username? That was laziness on my part haha.

Honestly the minimum IMO would be `username:uid:password` 🤷‍♂️  I have an old `awk` that you could potentially repurpose to this end, but the `chpass` pattern is probably better...: https://github.com/rstudio/sol-eng-docker/blob/41e51f4503788943afadb50e6bebdb3e478ddcc1/cluster/kerberos-rstudio/Dockerfile#L19 . `xargs` may be a better choice too.

I also think making these trivially unique would make it at least functional... A file like:
```
# username:uid:password
username1:1001:password
username2:1002:password
username3:1003:password
username4:1004:password
```
(then remove the comment with something like `grep -v '^\s*#'`? Just an idea 😄